tensorflow/python/ops/custom_gradient.py

```diff
@@ -62,7 +62,7 @@ def log1pexp(x):
   ```python
   x = tf.constant(100.)
   y = log1pexp(x)
-  dy_dx = tf.gradients(y, x) # Will be NaN when evaluated.
+  dy_dx = tf.GradientTape(y, x) # Will be NaN when evaluated.
```

Comment:
Thanks for the PR but this is not the right way to use the tf.GradientTape API. Please take a look at https://www.tensorflow.org/api_docs/python/tf/GradientTape. Mostly you need to open a GradientTape context.